core/src/main/java/org/apache/calcite/plan/RelTrait.java

```diff
@@ -91,6 +93,17 @@
   /**
    * Applies a mapping to this trait.
    *
+   * <p>Some traits may be changed if the columns order is changed by a mapping of the
+   * {@link Project} operator. </p>
+   *
+   * <p>For example, if relation {@code ORDER a, b} is sorted by columns [0, 1], then the
```

Comment:
Is it a typo? `ORDER a, b`

Reply:
Yes, this is a type. Fixed. Thank you.

Reply:
I see it is still there.

Reply:
I was sure I fixed it. Sorry. Now it's done.